client/src/test/java/io/pravega/client/control/impl/ControllerImplTest.java

```diff
@@ -1898,6 +1898,11 @@ public void testCreateKeyValueTable() throws Exception {
                 KeyValueTableConfiguration.builder().partitionCount(1).build());
         AssertExtensions.assertFutureThrows("Server should throw exception",
                 createKVTableStatus, Throwable -> true);
+
+        createKVTableStatus = controllerClient.createKeyValueTable("scope1", "kvtable6",
+                KeyValueTableConfiguration.builder().partitionCount(0).build());
+        AssertExtensions.assertFutureThrows("Server should throw exception",
+                createKVTableStatus, Throwable -> true);
```

Comment:
it is better to have a stronger predicate check which looks for expected exception

Reply:
Sure. Fixed.